src/renderer/components/_molecules/project-tree/index.tsx

```
@@ -120,7 +133,7 @@ const ProjectTreeBranch = ({ branchTarget, children, ...res }: ProjectTreeBranch
             'truncate font-caption text-xs font-normal text-neutral-850 dark:text-neutral-300',
             branchIsOpen && 'font-medium text-neutral-1000 dark:text-white',
           )}
-          dangerouslySetInnerHTML={{ __html: label || '' }}
+          dangerouslySetInnerHTML={{ __html: handleLabel(label) || '' }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)